### PR TITLE
Add alpha message

### PR DESCRIPTION
**Pull request by hev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Aerospike Backup Service
 
+```
+Aerospike Backup Service is currently in Alpha. Production usage is not reccomended and changes may occur.
```
Comment (arrowplum) [inline README.md]:
```suggestion
Aerospike Backup Service is currently in pre-alpha. Production usage is not recommended and changes to the API are likely to occur.
```

**Review by arrowplum — CHANGES_REQUESTED** [review]
Seems overkill to "request changes" but at least the spelling should be fixed.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # Aerospike Backup Service
 
+```
+Aerospike Backup Service is currently pre Alpha. Production usage is not reccomended and changes may occur.
```
Comment (arrowplum) [inline README.md:4]:
```suggestion
Aerospike Backup Service is currently pre Alpha. Production usage is not recommended and changes may occur.
```

**Review by arrowplum — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,5 +1,10 @@
 # Aerospike Backup Service
 
+```
+Aerospike Backup Service is currently pre Alpha. Production usage is not recommended and changes may occur.
+Enterprise customers, please contact support to sign an Alpha agreement. 
+```
```
Comment (reugn) [inline README.md:6]: I suggest using the [Alerts Markdown](https://github.blog/changelog/2023-12-14-new-markdown-extension-alerts-provide-distinctive-styling-for-significant-content/) extension
```
> [!NOTE]  
> Aerospike Backup Service is currently pre Alpha. Production usage is not recommended and changes may occur.
> Enterprise customers, please contact support to sign an Alpha agreement. 
```
and putting the message under the status badges.

**Review by reugn — APPROVED** [review]

**Pull request merged**